.github/labeler.yml

```diff
@@ -1,26 +1,34 @@
 # Add 'kernel' label to any change within Connectors, Extensions, Skills, and tests directories
 kernel:
-  - "dotnet/src/Connectors/**/*"
-  - "dotnet/src/Extensions/**/*"
-  - "dotnet/src/Skills/**/*"
-  - "dotnet/src/IntegrationTests/**/*"
-  - "dotnet/src/SemanticKernel.UnitTests/**/*"
+  - dotnet/src/**/*
```

Comment:
Can you test that this will not label changes that touch only  SemanticKernel.Abstractions as `kernel`?

Reply:
can we rename the labels? e.g. why would we exclude Abstractions from "kernel"?

Reply:
I'm open to renaming. I want a way to be able to identify whether a PR touches SemanticKernel or SemanticKernel.Abstractions, whether it touches Skills/Connectors/Extensions, or both.